extras/static_code_analysis.sh

```diff
@@ -50,6 +50,9 @@ PEP8_IGNORES="E121,E123,E126,E226,E24,E704"
 PEP8_IGNORES_EXAMPLES="${PEP8_IGNORES},E402"
 PEP8_IGNORES_TOPO_MANUAL="${PEP8_IGNORES_EXAMPLES},E265"
 
+# Drop files that should not be checked (space-separated list).
+FILES_TO_IGNORE="libnestutil/sparsetable.h"
```

Comment:
Files that are not under the NEST Initiative Copyright should be also added.
- compose.hpp
- hashtable-common.h
- libc_allocator_with_realloc.h
- sparseconfig.h
- sparsetable.h
- template_util.h
- type_traits.h